### ISSUE TITLE
[FR]: Support entry_points with output_dir = False

### ISSUE THREAD

**Issue by josephglanville** [aspect-build/rules_esbuild#155]
### What is the current behavior?

Currently when enabling either splitting or multiple entry-points then output_dir = True is enabled.

This is actually only necessary for the splitting case which will produce unpredictable output files, the multiple entry-point case will produce multiple files but those files will have predictable names based on the esbuild configuration.

### Describe the feature

Setting `entry_points` will correctly declare outputs for each of the bundled outputs and source maps etc and only switch to output directory declaration if splitting is enabled.

**Comment by Place1** [comment]
I think with support for this feature we might be able to use esbuild as a transpiler in rules_ts

**Comment by josephglanville** [comment]
Yeah it's essentially a requirement for use as a transpiler. I have this implemented on a branch. I will see if I can open a PR tomorrow or something after rebasing it.
